modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx

```diff
@@ -568,7 +568,7 @@ const PoliciesTable = () => {
 					policiesList.push({
 						commission: `$${commissionValue.toFixed(2)}`,
 						externalReferenceCode,
-						isClickable: 'true',
+						isClickable: productName === 'Auto' ? 'true' : 'false',
```

Comment:
It should be boolean ._.

Reply:
I changed it but in other places we have this problem.

Reply:
thanks sir!